src/components/SecondaryPanes/Expressions.css

```diff
@@ -43,8 +43,9 @@
 }
 
 .expression-container > .tree {
-  overflow: hidden;
+  overflow: auto;
```

Comment:
Thanks @arthur801031 I just tried replacing `overflow: auto` with `hidden` and it worked well.